stories/queryWizard/config.js

```diff
@@ -0,0 +1,111 @@
+import _ from 'lodash/fp'
+import { exampleTypes, mockService } from 'contexture-client'
+import ContextureMobx from '../../src/utils/contexture-mobx'
+
+let typeDescriptions = {
+  tagsQuery: {
+    description: 'Enter some tags.',
+  },
+  query: {
+    description: 'Enter a search term in the field.',
+  },
+  facet: {
+    description:
+      'Use the checkboxes to select all the results you wish to include. You can find and add multiple results by repeating this process.',
+  },
+}
+
+export let types = _.merge(exampleTypes, typeDescriptions)
+
+export let fields = {
+  foo: { label: 'Foo', description: 'Enter a foo.' },
+  bar: { label: 'Bar', description: "Type in the bar's name." },
+}
+
+let Client = ContextureMobx({
+  debug: true,
+  types,
+  service: mockService(),
+})
+
+export let tree = Client({
+  key: 'root',
+  join: 'and',
+  children: [
+    {
+      key: 'step 1',
+      label: 'Friendly Group',
+      type: 'group',
+      join: 'and',
+      children: [
+        {
+          type: 'tagsQuery',
+          key: 'friendly node',
+          label: 'Friendly Node',
+          typeDescription: 'Enter some tags',
+          fieldDescription: 'Search for stuff',
```

Comment:
These should never be on contexture nodes

Reply:
so let's take them out :)

Reply:
this one was pure oversight :(